### PR TITLE
go get cel.dev/exp experiment

### PR DESCRIPTION
**Pull request by sergiitk**
> [!CAUTION]
> Work in progress, do not merge.

### CODE REVIEW

**Review by l46kok — COMMENTED** [review]
Once PR is ready, could we regen these go files with 0.14? https://github.com/google/cel-spec/releases/tag/v0.14.0

File: go/expr/go.mod [hunk 1]
```diff
@@ -0,0 +1,8 @@
+module cel.dev/expr
```
Comment (l46kok) [inline go/expr/go.mod:1]: @TristonianJones Thoughts on the directory structure? In case if you wanted to serve any other generated golang protos from here

Comment (TristonianJones) [reply go/expr/go.mod:1]: I don't have a lot of opinions on the directory structure; however, I also want to cut back on Go's reliance on proto in general, so I think having this single directory is pretty perfect for making sure that continues to be the case.

**Comment by sergiitk** [comment]
Superseded by #341.

**Comment by sergiitk** [comment]
Correction: Superseded by #342.

**Pull request closed without merging**